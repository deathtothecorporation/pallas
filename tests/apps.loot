(K x y)=x
(S x y z)=((x z) (y z))
(I x)=(S K K x)

,, (0 1)
,, (1 2)
,, (3 3)
,, (4 4)
,, (5 5)
,, (I K (S I))
