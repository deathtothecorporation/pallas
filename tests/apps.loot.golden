(K a b)=a
(S a b c)=(a c b-c)
(I a)=(S K K a)
_=[0-1 1-2 4 0 0 (K S-I)]

_hash=(x#d3143353724a5ab77b27e2edd2741b9c0565b162c876c154e49795738ecad5c6)

